pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1907,6 +1907,34 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes[0].Field).To(Equal("fake.domain.devices.networkInterfaceMultiqueue"))
 		})
 
+		It("should accept vmi with a network multiqueue, with autoattached interfaces", func() {
```

Comment:
There is a pattern here, please try to use a table test.

Reply:
The difference between them is 
`vmi.Spec.Domain.Devices.AutoattachPodInterface = &var`
where &var equal either
nil
address of a real var

So i will need to create a struct for it with a pointer in it i think ? (if so sounds like overkill)
because i can't take the address of nil (and nil is needed for the first one)

Do you have other nicer suggestion maybe please? (i can use a condition as well, if you are fine with it)